src/methods/include.ts

```diff
@@ -0,0 +1,18 @@
+import type { FramebusConfig } from "../framebus-config";
+import { childWindows } from "../lib/constants";
+
+export function include(config: FramebusConfig, childWindow: Window): boolean {
```

Comment:
It doesn't look like the config is used at all.

Reply:
Yeah, you're right, but we wanted to keep API the same for all the methods an integrator would use. That way if in the future, if we need something from the config, we can easily update this method without making the integrator change their integration or handle multiple types for the arguments. (IE, checking if the first arg is a window or a FramebusConfig)

Reply:
Ah got it. All the others work with a config as the first arg and so you're matching that paradigm even if this module does not yet use config.